src/main/java/duke/Event.java

```diff
@@ -0,0 +1,39 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * <h>duke.Event duke.Task Type</h>
+ * This is a type of tasks that start at a specific time and ends at a specific time
+ */
+public class Event extends Task {
+    protected LocalDate at;
+
+    /**
+     * Constructor of duke.Event class.
+     * @param description Name of the task input by user.
+     * @param at Time of the task input by user.
+     */
```

Comment:
Good use of javadocs again. Probably a bit more description might be needed but otherwise it is good practise to have documentation on almost all your non trivial methods and constructors.